spec/controllers/api/v1/accounts/macros_controller_spec.rb

```diff
@@ -249,7 +249,6 @@
     context 'when it is an authenticated user' do
       it 'execute the macro' do
         expect(conversation.messages).to be_empty
-        expect(conversation.assignee).to be_nil
```

Comment:
@tejaswinichile Why was this line removed?

Reply:
Oh because it was failing in rubocop :D Too many lines in one spec.

Reply:
Added the specs here for sender check https://github.com/chatwoot/chatwoot/pull/5769/files#diff-05bb0b4908bb3017214fab118b732e88f575364d2ed97930bbec973d4c70a49eR261